modules/lead/toolchain/kritis.tf

```diff
@@ -0,0 +1,65 @@
+module "kritis_certificate" {
+  source = "../../common/certificates"
+
+  enabled = true
+  name = "kritis-cert"
+  namespace = var.namespace
+  domain = "kritis-validation-hook.toolchain.svc"
```

Comment:
The namespace name toolchain is configurable via var.namespace...can you use that instead?

Reply:
Fixed to use namespace var.